frontend/src/pages/home/index.html

```diff
@@ -53,21 +58,30 @@
           <!-- message box -->
 
           <div
-            class="mt-[52px] pb-36 px-[15px] overflow-y-auto z-20 max-h-screen scrollbar md:w-full snap-y"
+            class="mt-[52px] pb-36 px-[15px] z-50 overflow-y-auto max-h-screen scrollbar md:w-full snap-y"
```

Comment:
when the screen is full with messages the scrollbar is not in the right place. it goes up a little bit.